clients/src/test/java/org/apache/kafka/common/security/authenticator/ClientAuthenticationFailureTest.java

```diff
@@ -85,12 +86,12 @@ public void testConsumerWithInvalidCredentials() {
 
         try (KafkaConsumer<String, String> consumer = new KafkaConsumer<>(props, deserializer, deserializer)) {
             consumer.subscribe(Arrays.asList(topic));
-            consumer.poll(100);
+            consumer.poll(Duration.ofSeconds(10));
```

Comment:
Time actually does pass during this test, so we need the timeout to account for the metadata update. On my machine, this test takes ~800ms to complete, so I figure 10s should be enough elbow room.